### PR TITLE
avoid misleading message

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

File: src/ipv4.rs [hunk 1]
```diff
@@ -309,6 +309,12 @@ mod test {
     }
 
     #[test]
+    fn parse_v4_fail_2slashes() {
```
Comment (achanda) [inline src/ipv4.rs]: Would you mind changing this to `parse_v4_fail_two_slashes`, please?

File: src/ipv6.rs [hunk 2]
```diff
@@ -293,6 +293,12 @@ mod test {
     }
 
     #[test]
+    fn parse_v6_fail_2slashes() {
```
Comment (achanda) [inline src/ipv6.rs]: `two` here

**Comment by tshepang** [comment]
done

File: src/ipv4.rs (hunk 1 above)
Comment (tshepang) [reply src/ipv4.rs]: done

File: src/ipv6.rs (hunk 2 above)
Comment (tshepang) [reply src/ipv6.rs]: also done

**Pull request merged**